tests/factory_fixtures/property.py

```diff
@@ -23,10 +23,10 @@ def _property_attributes(archived=False, manager_ids=None):
 
 
 @pytest.fixture
-def create_property(property_attributes):
-    def _create_property():
+def create_property(property_attributes, **kwargs):
```

Comment:
This is very nice. I don't think we need the Kwargs here though.

Reply:
I put the **kwargs here so I could pass `manager_ids` directly into the `create_property` fixture in the tests.  See line 108 of `tests/integration/test_users.py`